src/cmd/src/cli/export.rs

```diff
@@ -322,15 +327,25 @@ impl Export {
                     .await
                     .context(FileIoSnafu)?;
                 let output_dir = Path::new(&self.output_dir).join(format!("{catalog}-{schema}/"));
-
-                // copy database to
-                let sql = format!(
-                    "copy database {} to '{}' with (format='parquet');",
-                    schema,
-                    output_dir.to_str().unwrap()
-                );
-                self.sql(&sql).await?;
-                info!("finished exporting {catalog}.{schema} data");
+                // Ignores metric physical tables
```

Comment:
nit: maybe it makes sense to rename `metrics_tables` to `metric_physical_tables` to be consistent with the above code and the comment.

Reply:
Good catch!